services/auth/office/constants.js

```diff
@@ -0,0 +1,15 @@
+const JWK = {
+  USE: 'sig',
+  KTY: 'RSA',
+};
+
+const JWT = {
+  ALG: 'RS256',
+};
+
+const loginMicrosoftUrl = 'https://login.microsoftonline.com';
+
+const beginCertificateString = '-----BEGIN CERTIFICATE-----\n';
+const endCertificateString = '\n-----END CERTIFICATE-----\n';
```

Comment:
Is there a reason for not exporting the const directly? ie 
```JS
export const ...
export const ...
export const ...
```
Instead of 
```JS
export { constant, ...}
```

Reply:
More like having a common place of exporting all constants from. But Im fine either way.